src/index.tsx

```diff
@@ -1,99 +1,19 @@
-import React, { FC, ReactNode, useEffect, useRef } from "react";
+import { ReactNode, useEffect, useRef } from "react";
 import * as ReactDOM from "react-dom";
-import { ChatView } from "components/ChatView";
+import { Props, ChatView } from "./components/ChatView";
+import type { SummaryLanguage } from "./types";
+export type { Props } from "components/ChatView";
+export { DEFAULT_SUMMARIZER } from "./useChat";
 
 // @ts-ignore
 import cssText from "index.scss";
-import type { SummaryLanguage } from "./types";
-
-export interface Props {
```

Comment:
I moved this to `ChatView.tsx` so that file's role is to define the chatbot interface, and this file's role is to act as a mediation layer between the non-React and React environments.